Automattic/wp-calypso
# Plans: Update Enterprise hardcoded pricing to use formatCurrency

This will pass the hardcoded `$25,000` price point for Enterprise plans through formatCurrency to ensure the formatting is localized and displayed correctly across languages/locales.

Related to #74605

## Testing Instructions

* Set your user currency to EUR (or any non-USD currency)
* Set locale to `fr-CA` from `/me` for French Canadian (Espanol works too)
* Go to `/plans/[your-site-name]`

If you're within the US, you should see the following:
![image](https://user-images.githubusercontent.com/16580129/226776894-8c80f9f6-e410-4a60-837c-df9190ec000f.png)

If you're outside the US, you should see:
![image](https://user-images.githubusercontent.com/16580129/226777039-7a925fd1-6fd2-421c-a4b6-8462a697ffbd.png)

Note that the currencies are showing as right aligned, this is how formatCurrency aligns for `fr-CA` and `es` locales. Also note that when outside of the US you'll see `US$` right aligned, this is expected and correct. 

You may also run into some weird fringe cases if you're in Canada or Mexico where the currency is showing as `$`, this is because you're actually hitting our US servers, in those cases you may need to use a VPN to test this.
